src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/BuildUpToDateCheck.cs

```diff
@@ -46,16 +53,14 @@ private void Log(LogLevel level, string message, params object[] values)
 
         private const string TrueValue = "true";
         private const string FullPath = "FullPath";
-        private const string ResolvedPath = "ResolvedPath";
         private const string CopyToOutputDirectory = "CopyToOutputDirectory";
-        private const string Never = "Never";
-        private const string OriginalPath = "OriginalPath";
+        private const string PreserveNewest = "PreserveNewest";
+        private const string Always = "Always";
 
```

Comment:
you can say something like None.CopyToOutputDirectoryValues.PreserveNewest. This is auto-generated from the None.xaml rule.